packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -255,7 +255,7 @@ def get_addresses(self) -> Generator:  # pylint: disable=too-many-return-stateme
             return False
 
         # put service info in the shared state for p2p message handler
-        info: Dict[str, Dict[str, str]] = dict.fromkeys(registered_addresses)
+        info: Dict[str, Dict[str, str]] = {i: {} for i in registered_addresses}
```

Comment:
why the change?

Reply:
@DavidMinarsch to create default value as `{}`, not `None`

Reply:
```suggestion
        info: Dict[str, Dict[str, str]] = dict.fromkeys(registered_addresses, {})
```